docs/section12/Rule12-3.md

```diff
@@ -1,38 +1,58 @@
-
 # Receptacle - Rule 12-3
-
+**Schema Version** 0.0.37  
+**Primary Rule:** True  
 **Rule ID:** 12-3  
-**Rule Description:** User RMR Space ID in Proposed RMR?  
-**Rule Assertion:** Proposed RMR = User RMR  
-**Appendix G Section:** Section Table G3.1-12 Receptacles: Modeling Requirements for the Proposed design  
+**Rule Description:** When receptacle controls installed in spaces where not required by Standard 90.1 2019, Section 8.4.2 are included in the proposed building design, the hourly receptacle shall be reduced as specified in Standard 90.1-2019, Table G3.1 Section 12, Proposed Building Performance.   
+**Appendix G Section:** Table G3.1 Section 12 Baseline Building Performance  
 **Appendix G Section Reference:** None  
 
-**Applicability:** All required data elements exist for P_RMR and U_RMR  
-**Applicability Checks:**  
-
-  1. User RMR Space Count > 0  
+**Applicability:** All required data elements exist for B_RMR and P_RMR  
+**Applicability Checks:**
+1. Receptacle controls are installed in spaces where not required by Standard 90.1 2019, Section 8.4.2  
 
 **Manual Check:** None  
-**Evaluation Context:** Each Data Element  
+**Evaluation Context:** Each Miscellaneous Equipment object in the proposed model    
 **Data Lookup:** None  
+**Function Call:**
+1) get_component_by_id
+2) compare_schedules
+
+## Applicability Checks:  
+- Create a list of the lighting space types that correspond to space types that may have receptacle control requirements in Section 8.4.1: `EXPECTED_RECEPTACLE_CONTROL_SPACE_TYPES = ["OFFICE_ENCLOSED", "CONFERENCE_MEETING_MULTIPURPOSE_ROOM", "COPY_PRINT_ROOM", "LOUNGE_BREAKROOM_HEALTH_CARE_FACILITY", "LOUNGE_BREAKROOM_ALL_OTHERS", "CLASSROOM_LECTURE_HALL_TRAINING_ROOM_PENITENTIARY", "CLASSROOM_LECTURE_HALL_TRAINING_ROOM_SCHOOL", "CLASSROOM_LECTURE_HALL_TRAINING_ROOM_ALL_OTHER", "OFFICE_OPEN_PLAN"]`  
+- Create a list to store the spaces that have receptacle controls installed where not required by Standard 90.1 2019, Section 8.4.2: `spaces_with_receptacle_controls_beyond_req = []`  
+- Iterate through the spaces in the proposed model: `for space_p in P_RMD...spaces`  
+  - Get the lighting space type: `space_type_p = space_p.lighting_space_type`  
+  - Iterate through the miscellaneous equipment loads in the space: `for misc_equip_p in space_p.miscellaneous_equipment:`  
+    - Get the proposed automatic receptacle control: `auto_receptacle_control_p = misc_equip_p.has_automatic_control`  
+    - If the space type is not in the list of space types where receptacle controls may be required, and the space has automatic receptacle controls installed: `if space_type_p not in EXPECTED_RECEPTACLE_CONTROL_SPACE_TYPES and auto_receptacle_control_p:`  
```

Comment:
not sure why line 27 is 'may be required', should this be is required? are we saying 'may be' because the requirement only applies to 125V, 15 and 20 amp receptacles?

Reply:
Exactly